### PR TITLE
222

### PR DESCRIPTION
**Pull request by Dino153**

### CODE REVIEW

**Review by Kurash1 — APPROVED** [review]
LGTM

File: src/random_events/estate_orcs.txt [hunk 1]
```diff
@@ -608,4 +609,123 @@ new country_event orc_estate_events.20 = {
...
+	inherit = random_event
+	title = "We have Died."
+	desc = "Unfortunately, we were unsuccesful in defeating the Old Orc, this has resulted in our death."
+	picture = loose_berserker_eventPicture
+	goto = old_orc_province
+	options = {
+		{
+			name = "Curses!"
+			effect = {
+				kill_ruler = yes
```
Comment (JoMakka) [inline src/random_events/estate_orcs.txt:727]: damn ,that harsh?

File: src/functions/ruler_skill_check.txt [hunk 2]
```diff
@@ -0,0 +1,44 @@
+new effect ruler_skill_check = {
+	args = {
+		string type
```
Comment (JoMakka) [inline src/functions/ruler_skill_check.txt:3]: unfortunately i can't understand this part yet so i can't say anything about it kurashi check this file again to be sure

File: src/random_events/estate_orcs.txt [hunk 3]
```diff
@@ -608,4 +609,123 @@ new country_event orc_estate_events.20 = {
...
+			}
+		}
+	}
+}
```
Comment (JoMakka) [inline src/random_events/estate_orcs.txt:731]: this file looks good to me otherwise

File: src/functions/ruler_skill_check.txt (hunk 2 above)
Comment (Kurash1) [reply src/functions/ruler_skill_check.txt:3]: I wrote it.

**Pull request merged**